docs/data-sources/rdb_database.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: "scaleway"
+page_title: "Scaleway: scaleway_rdb_database"
+description: |-
+  Gets information about an RDB database.
+---
+
+# scaleway_rdb_instance
+
+Gets information about a RDB database.
+
+## Example Usage
+
+```hcl
+# Get the database foobar hosted on instance id 11111111-1111-1111-1111-111111111111
+data "scaleway_rdb_database" "my_db" {
+  instance_id = "11111111-1111-1111-1111-111111111111"
+  name        = "foobar"
+}
+# Find the first database hosted on instance id 11111111-1111-1111-1111-111111111111
```

Comment:
Is it not deterministic? Should the name be mandatory?

Reply:
I can make it mandatory sure.

Reply:
https://github.com/remyleone/terraform-provider-scaleway/pull/4